examples/hackernews/web_service/app.py

```diff
@@ -47,7 +47,7 @@ def posts_index():
     if 'text/event-stream' in request.accept_mimetypes:
         resp = requests.post(
             f"{REACTIVE_SERVICE_URL}/streams/posts",
-            json={ 'limit': 10 },
+            json=10,
```

Comment:
Maybe it would be clearer to update the [`PostsResource` constructor](https://github.com/SkipLabs/skip/blob/fd42b0158a50f9437521fbfc98adc12b4cbd0ea5/examples/hackernews/reactive_service/src/hackernews.service.ts#L93) to properly deconstruct the `params` instead? Up to you, this is fine as well.